packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -18,6 +18,7 @@ import styled, { ThemeContext } from 'styled-components';
 import { ApiPromise } from '@polkadot/api';
 import { AddressInfo, AddressMini, AddressSmall, Badge, Button, ChainLock, CryptoType, Forget, Icon, IdentityIcon, LinkExternal, Menu, Popup, StatusContext, Tags } from '@polkadot/react-components';
 import { useAccountInfo, useApi, useBestNumber, useCall, useLedger, useToggle } from '@polkadot/react-hooks';
+import { useBalancesAll } from '@polkadot/react-hooks/useBalancesAll';
```

Comment:
Combine with above import. I understand this is done for mocking purposes, however it is a bit diffcult to have to code around mocks, rather they should adapt.

Reply:
Done